controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -382,7 +289,7 @@ func recordActionFailure(info *reconcileInfo, errorType metal3v1alpha1.ErrorType
 	return actionFailed{dirty: true, ErrorType: errorType, errorCount: info.host.Status.ErrorCount}
 }
 
-func (r *ReconcileBareMetalHost) credentialsErrorResult(err error, request reconcile.Request, host *metal3v1alpha1.BareMetalHost) (reconcile.Result, error) {
+func (r *BareMetalHostReconciler) credentialsErrorResult(err error, request reconcile.Request, host *metal3v1alpha1.BareMetalHost) (ctrl.Result, error) {
```

Comment:
Yeah, I was trying to be consistent but I guess I missed some. I wanted to rename everything in the patch tied to the kubebuilder output, since that's what triggered the rename. I could split that part of the commit out, though, if you think that would make the history clearer?

Reply:
Fixed.